suricata.yaml.in

```diff
@@ -1055,7 +1055,8 @@ vlan:
 # of that flow). The value of "established" is the amount of
 # seconds that the engine will wait to free the flow if it spend that amount
 # without receiving new packets or closing the connection. "closed" is the
-# amount of time to wait after a flow is closed (usually zero).
+# amount of time to wait after a flow is closed (usually zero). "bypassed" is
```

Comment:
Not sure this helps. I think it should explain wait the purpose is. Here one just wonders: if equivalent to established why does it exist?